### PR TITLE
Pager UI updates

### PR DESCRIPTION
**Pull request by marinaaisa**
Bug/issue #122817341, if applicable: 

## Summary

Updates the UI in the medium viewport:

1. There should only be 2 rows max
2. The pagination dots should disappear in this viewport
3. Both of the chevrons should appear at the same time, and one of them should be disabled if there's no more cards in that "direction"

It also moves the `static-card-size` var into a global scope.

## Checklist

Make sure you check off the following items. If they cannot be completed, provide a reason.

- [x] Added tests
- [x] Ran `npm test`, and it succeeded
- [x] Updated documentation if necessary

### CODE REVIEW

**Review by SamLanier — APPROVED** [review]
Design approved.

**Comment by marinaaisa** [comment]
@swift-ci test

File: tests/unit/components/DocumentationTopic/TopicsLinkCardGrid.spec.js [hunk 1]
```diff
@@ -56,11 +56,11 @@ describe('TopicsLinkCardGrid', () => {
       expect(pager.exists()).toBe(true);
       expect(pager.props('pages').length).toBe(2);
 
-      // 10 items => 3 pages at medium breakpoint (4 links per page)
+      // 10 items => 2 pages at large breakpoint (6 links per page)
```
Comment (hqhhuang) [inline tests/unit/components/DocumentationTopic/TopicsLinkCardGrid.spec.js:59]: This comment should probably still say `medium`?

**Review by hqhhuang — APPROVED** [review]

File: tests/unit/components/DocumentationTopic/TopicsLinkCardGrid.spec.js (hunk 1 above)
Comment (marinaaisa) [reply tests/unit/components/DocumentationTopic/TopicsLinkCardGrid.spec.js]: Done! Good catch ;) https://github.com/apple/swift-docc-render/pull/787/commits/e841161bde0f5b83f612ccc0aa34e14b72354044

**Comment by marinaaisa** [comment]
@swift-ci test

**Pull request merged**